bench/config/nginx.py

```diff
@@ -1,8 +1,16 @@
 import os, json, click, random, string, hashlib
 from bench.utils import get_sites, get_bench_name, exec_cmd
 from six import string_types
+from six.moves import input
+
 
 def make_nginx_conf(bench_path, yes=False):
+	conf_path = os.path.join(bench_path, "config", "nginx.conf")
+
+	if not yes and os.path.exists(conf_path):
+		if input('nginx.conf already exists and this will overwrite it. Do you want to continue? [y/N]').lower() == 'n':
```

Comment:
instead of this we can still use click without `abort=True` and then indent all the code inside this.

anything that is not 'n' will still be rendered as a yes with this.

Reply:
Something like https://github.com/frappe/bench/commit/ba90c9fd7c919add30ee94f9e3f85540d26948c0 but with `click.confirm`?

Reply:
https://click.palletsprojects.com/en/7.x/prompts/#confirmation-prompts

so something like: 
```python3
if click.confirm("Press enter to continue: "):
	setup_nginx()
	...
```